pallets/allocations/src/weights.rs

```diff
@@ -57,106 +57,68 @@ pub trait WeightInfo {
 
 /// Weights for pallet_allocations using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
+
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	// Storage: Allocations SessionQuota (r:1 w:1)
-	// Storage: Balances TotalIssuance (r:1 w:1)
-	// Storage: System Account (r:3 w:3)
-	// Storage: System Number (r:1 w:0)
-	// Storage: System ExecutionPhase (r:1 w:0)
-	// Storage: System EventCount (r:1 w:1)
-	// Storage: System Events (r:1 w:1)
-	/// The range of component `b` is `[1, 500]`.
```

Comment:
Don't we have this informative comments in the new weight files?

Reply:
I'm not sure where that comes from, this file is supposed to be auto-generated.